po/wesnoth-lib/de.po

```diff
@@ -8605,8 +8497,6 @@ msgstr "Keine Liste definiert."
 
 #: src/gui/widgets/listbox.cpp:774 src/gui/widgets/listbox.cpp:817
 #: src/gui/widgets/listbox.cpp:860
-#, fuzzy
-#| msgid "A 'list_definition' should contain one row."
 msgid "A ‘list_definition’ should contain one row."
 msgstr "Eine »list_definition« sollte nur eine Zeile enthalten."
```

Comment:
Noticed in the German, but should be changed in the English: it "should contain *exactly* one row". The message looks like it will be shown for zero rows. too.